metpy/plots/tests/test_ctables.py

```diff
@@ -136,3 +137,10 @@ def test_gempak():
     result = outfile.read()
 
     assert result == '(0.000000, 0.000000, 0.000000)\n(1.000000, 1.000000, 1.000000)\n'
+
+
+@check_and_silence_deprecation
+def test_viridis(registry):
```

Comment:
Argh. Can't use our nice decorator and a test fixture (i.e. `registry`) on Python 2.7. Need to catch/check the warning by hand. 😭